src/components/ArtworksList/ArtworkLargeCard.vue

```diff
@@ -54,7 +54,7 @@ h3
   display: block
   border: 1px solid #eee
   width: 240px
-  max-width: calc(45vw - 1.5rem)
+  max-width: calc(50vw - 32px)
```

Comment:
16px 是 gap，不是 margin

Reply:
原来这里写的是2rem，是改回去还是16px

Reply:
https://github.com/FreeNowOrg/PixivNow/blob/52ec2190460bf7cb70cb99272c535d1a1f03b249/src/components/ArtworksList/ArtworkLargeList.vue#L8

没记错的话这里的 gap 确实是 16px
```suggestion
  max-width: calc(50vw - 32px)
```

Reply:
gap当然确实是16px，我只是想确定这里就写-16px是否合适
不过如果宽度直接就是50vw-16px的话好像会溢出